packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -1196,9 +1196,18 @@ export class SnapController extends BaseController<
     this.messagingSystem.publish(`${controllerName}:snapUnblocked`, snapId);
   }
 
+  /**
+   * Assert that installation of a given Snap is allowed based on the allow-list requirements and permissions.
+   *
+   * @param snapId - ID of a Snap.
+   * @param snapInfo - An object containing basic snap information including initial and dynamic permissions.
+   */
   async #assertIsInstallAllowed(
     snapId: SnapId,
-    snapInfo: SnapsRegistryInfo & { permissions: SnapPermissions },
+    snapInfo: SnapsRegistryInfo & {
+      permissions: SnapPermissions;
```

Comment:
```suggestion
      initialPermissions: SnapPermissions;
```
Let's distinguish now!

Reply:
Updated. 
Note: minor adjustments to the params had to be made here: https://github.com/MetaMask/snaps/blob/43fc176bd1e6170af1f9a4ccb2258f8084e97fb6/packages/snaps-controllers/src/snaps/SnapController.ts#L1213 and the test is updated here:  https://github.com/MetaMask/snaps/blob/43fc176bd1e6170af1f9a4ccb2258f8084e97fb6/packages/snaps-controllers/src/snaps/SnapController.test.ts#L590
Passing `permissions` were redundant in any way, right?